src/index.js

```diff
@@ -844,6 +844,10 @@ class JanusAdapter {
   }
 
   enableMicrophone(enabled) {
+    if (this.reenableTimeout) {
```

Comment:
in the case where we are replacing the video track (eg screen sharing) won't this end up cancelling the workaround setTimeout?

this bug only manifests in firefox with video streams, so perhaps in the interest of keeping it simple, we can only apply it there (and assume we'll remove it once the fix lands in stable)

Reply:
:+1: